.NET/Microsoft.Recognizers.Text.DateTime/Japanese/Extractors/DateTimeExtractorJap.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System.Linq;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Japanese;
+using DateObject = System.DateTime;
+using Microsoft.Recognizers.Text.Number;
+
+namespace Microsoft.Recognizers.Text.DateTime.Japanese
+{
+    public class DateTimeExtractorJap : IDateTimeExtractor

```

Comment:
There is no need to add the Jap suffix. Could you rename classes to remove the suffixes?
I see that some classes in Chinese use it. We'll remove those too.